File: src/mffs/container/ContainerBiometricIdentifier.java
package mffs.container;

import mffs.base.ContainerBase;
import mffs.slot.SlotActive;
import mffs.slot.SlotBase;
import mffs.tileentity.TileEntityBiometricIdentifier;
import net.minecraft.entity.player.EntityPlayer;

public class ContainerBiometricIdentifier extends ContainerBase
{
	public ContainerBiometricIdentifier(EntityPlayer player, TileEntityBiometricIdentifier tileentity)
	{
		super(tileentity);

		this.addSlotToContainer(new SlotActive(tileentity, 0, 88, 91));

		this.addSlotToContainer(new SlotBase(tileentity, 1, 8, 46));
		this.addSlotToContainer(new SlotActive(tileentity, 2, 8, 91));

		for (int var4 = 0; var4 < 9; var4++)
		{
			this.addSlotToContainer(new SlotActive(tileentity, 3 + var4, 8 + var4 * 18, 111));
		}

		this.addSlotToContainer(new SlotBase(tileentity, TileEntityBiometricIdentifier.SLOT_COPY, 8, 66));

		this.addPlayerInventory(player);
	}

}

File: src/main/scala/mffs/security/ContainerBiometricIdentifier.scala
package mffs.security

import mffs.item.card.ItemCardFrequency
import mffs.slot.SlotBase
import net.minecraft.entity.player.EntityPlayer
import resonant.lib.gui.ContainerBase
import resonant.lib.prefab.slot.SlotSpecific

class ContainerBiometricIdentifier(player: EntityPlayer, tile: TileBiometricIdentifier) extends ContainerBase(tile)
{
  //Frequency
  addSlotToContainer(new SlotSpecific(tile, 0, 8, 114, classOf[ItemCardFrequency]))

  for (x <- 0 until 9; y <- 0 until 4)
    addSlotToContainer(new SlotBase(tile, x + y * 9 + 1, 9 + x * 18, 36 + y * 18))

  addPlayerInventory(player)
}